internal/tokens/tifactory/factory.go

```diff
@@ -30,6 +30,16 @@ var pluginsByName = map[string]func() tokens.Plugin{
 	(*fftokens.FFTokens)(nil).Name(): func() tokens.Plugin { return &fftokens.FFTokens{} },
 }
 
+func InitPrefixWithSubPrefix(prefix config.PrefixArray) {
+	prefix.AddKnownKey(tokens.TokensConfigName)
+	prefix.AddKnownKey(tokens.TokensConfigType)
+	for name, plugin := range pluginsByName {
+		// Accept a superset of configs allowed by all plugins
```

Comment:
This comment is inaccurate now (we aren't squashing the config into a superset as before)